lib/services/lwm2mHandlers/registration.js

```diff
@@ -188,10 +188,20 @@ function addUnsupportedAttributes(payload, configuration, deviceInformation, cal
             if (omaRegistry[value.objectType].resources) {
                 return previous.concat(omaRegistry[value.objectType].resources);
             } else {
-                previous.push({
-                    name: omaRegistry[value.objectType].name + '#' + value.objectInstance,
-                    type: 'string'
-                });
+
+                // In NGSIv2, spaces cannot be used in attributes' names. Therefore, we trim them.
```

Comment:
Taking into account how it has been implemented at the end, I understand "... Therefore, we trim them" should be changed.

Reply:
Fixed with 0196c4629d6b73640d7bd26a0b183ecc80e2f677